src/zstream.c

```diff
@@ -87,9 +87,51 @@ size_t git_zstream_suggest_output_len(git_zstream *zstream)
 		return ZSTREAM_BUFFER_MIN_EXTRA;
```

Comment:
There's a typo in the commit message: `git_output` should be `get_output`.

Reply:
Fixed.